src/modules/verifiable-credential-storage/entities/vc-storage.entity.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  CredentialSubject,
+  VerifiableCredential,
+} from '@ew-did-registry/credentials-interface';
+import { Column, PrimaryGeneratedColumn, Entity } from 'typeorm';
+import { IVCStorage } from '../types/vc-storage';
+
+@Entity()
+export class VCStorage implements IVCStorage {
```

Comment:
IMO, this module shouldn't really know anything about VCs. I would therefore change this entity to be a `CollectionsObject` and just have properties that are generic to `CollectionsObjects` (`schema`, `dataFormat`, `dateCreated`, `datePublished`) https://identity.foundation/decentralized-web-node/spec/#write

In the case of VCs, I would then set the properties as:
- `schema`: An array containing the `types` of the VCs https://www.w3.org/TR/vc-data-model/#types
- `dataFormat`: `application/json` or `application/ld+json` (if it is a JSON-LD VC https://www.w3.org/TR/json-ld/#interpreting-json-as-json-ld)
- `dateCreated`: the UNIX Epoch timestamp that the object was initially stored
- `datePublished`: I think we just leave this blank for now?